packages/select/src/Option.tsx

```diff
@@ -17,6 +18,7 @@ const optionStyle = css`
   display: flex;
   width: 100%;
   padding: 10px 12px;
+  font-family: ${fontFamilies.default};
```

Comment:
Why are we adding the font-family declaration on the options and option groups? We should probably move the font-family declaration to the dropdown container itself (the root child of the portal).

Reply:
👍